### PR TITLE
Story/14836 remove global width constraints

### PR DESCRIPTION
**Pull request by gm3197**
* Remove global p `max-width`
* Use Grid to fix overflowing as a result of ^

### CODE REVIEW

File: pages/impact/index.tsx [hunk 1]
```diff
@@ -593,7 +593,7 @@ const Index = ({
                 >
                   <Grid container marginCenter id="public">
```
Comment (michaeljwiebe) [inline pages/impact/index.tsx]: second id="public" on this one

File: pages/impact/index.tsx [hunk 2]
```diff
@@ -294,7 +294,7 @@ const Index = ({
                 >
                   <Grid container marginCenter id="defense">
                     <Grid row className={Style.defensesContent}>
-                      <Grid size={{ Xs: 12 }}>
+                      <Grid size={{ Xs: 12, Lg: 3 }}>
```
Comment (michaeljwiebe) [inline pages/impact/index.tsx]: to match the current site i think this should be 4 at least then Xl: 3 is ok

Comment (michaeljwiebe) [reply pages/impact/index.tsx]: thats all the others below it too

**Review by michaeljwiebe — CHANGES_REQUESTED** [review]
see comments

File: pages/impact/index.tsx (hunk 2 above)
Comment (michaeljwiebe) [reply pages/impact/index.tsx]: did you change this? or is there a problem?

Comment (gm3197) [reply pages/impact/index.tsx]: it's _actually_ done this time

**Review by michaeljwiebe — APPROVED** [review]

**Pull request merged**